.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: bash
+services: docker
```

Comment:
Doesn't this require `sudo` as well? Otherwise you won't be able to talk to docker iirc.

Reply:
The problem doesn't seem to be related to that, but rather the build log being too long. It seems the log is too long. I tried separating the build and the run, but the run alone seeems to be too long.
I'm trying things on https://github.com/jtimon/lightning-integration/pull/11
See https://api.travis-ci.com/v3/job/220260237/log.txt for a failing log.